lib/src/shared/main/java/com/couchbase/lite/internal/core/C4Replicator.java

```diff
@@ -247,6 +249,22 @@ public C4ReplicatorStatus getStatus() {
         }
     }
 
+    public Set<String> getPendingDocIDs() throws LiteCoreException {
+        synchronized (lock) {
+            if (handle == 0) { return null; }
+
+            FLSliceResult result = new FLSliceResult(getPendingDocIds(handle));
+
+            return null;
```

Comment:
wait, this is empty? is always going to return null?

Reply:
no.  This is not complete.